website/docs/pipelines/promoting-applications.mdx

```diff
@@ -140,118 +142,170 @@ is upgraded in one of the pipeline's environments, a PR is created that updates
 The dynamic nature of GitOps deployments requires you to assist Weave GitOps a little with information on which repository hosts the manifest files,
 how to authenticate with the repository and the Git provider API, and which file hosts the version definition for each environment.
 
-:::caution
+### Security
+
+:::danger
 
-Creating pull requests requires read and write access in your git repo.
-A compromised token could lead to changing the git repository and creation of legitimate looking changes/pull requests.
+Create pull requests requires write access to your git repo. In case your pull request secret is compromised (and it is
+suggested you assume it as a likely scenario more than unlikely), it could end up for example, in deleting all your
+production applications.
 
-Ensure you understand and adopt [security recommendations](./#security-recommendations) before using the feature.
+This section should not be read as best practices but as baseline to enable automated pull requests in lower environments.
 
+It targets platform admins before enabling the capability.
 :::
 
-### Supported Git Providers
-The following Git providers are currently support by this promotion strategy:
+#### Environments
 
-- [GitHub](https://github.com/)
-- [GitLab](https://gitlab.com/)
-- [BitBucket Server / DataCenter](https://www.atlassian.com/software/bitbucket/enterprise)
+1. **Use it in a lower environment when there are no attack vectors to production**: avoid using it if a compromised token could
+make changes in a lower environment (for example test or staging) that could be promoted to production being undetected
+nor rejected.
 
-Select your Git provider via `.spec.promotion.strategy.pull-request.type`. For example, for `gitlab` it would look similar to:
+#### RBAC
 
-```yaml
-promotion:
-  strategy:
-    pull-request:
-      type: gitlab
-      url: "https://gitlab.com/weaveworks/<my-awesome-project.git>"
-      baseBranch: main
-      secretRef:
-        name: gitlab-promotion-credentials
-```
+You have configured Least privilege RBAC following [Good Practices](https://kubernetes.io/docs/concepts/security/rbac-good-practices/).
+Emphasis in the following:
 
-More info in the [spec](../spec/v1alpha1/pipeline/#pipeline).
+> Assign permissions at the namespace level where possible. Use RoleBindings as opposed to ClusterRoleBindings
+> to give users rights only within a specific namespace.
 
-### Credentials Secret
+> Avoid providing wildcard permissions when possible, especially to all resources.
+> As Kubernetes is an extensible system, providing wildcard access gives rights not just to
+> all object types that currently exist in the cluster, but also to all object types which are created in the future.
 
-In the journey of creating a pull request, there are different secrets involved:
+Avoid list and watch on secrets as:
```

Comment:
do not extract get of the equation

Reply:
what about the scenario where granting get access to a service account and someone init he namespace can run pods with that service account

Reply:
>what about the scenario where granting get access to a service account and someone init he namespace can run pods with that service account
added